arch/x86/core/pcie.c

```diff
@@ -206,6 +210,7 @@ uint32_t pcie_msi_map(unsigned int irq,
 uint16_t pcie_msi_mdr(unsigned int irq,
 		      msi_vector_t *vector)
 {
+  uint16_t msg_data_reg = 0;
```

Comment:
The indentation seems wrong here (spaces instead of a tab). Also, for the most part the coding style recommends an empty line after variable declarations.